webapp/plugins/authPlugin.js

```diff
@@ -0,0 +1,4 @@
+export default ( { app, store }) => {
+    const token = app.$apolloHelpers.getToken();
```

Comment:
This doesn't work in SSR due to a bug. I put a hint in the exercise description.